tests/phpunit/elementor/data/base/mock/processor/controller.php

```diff
@@ -1,11 +0,0 @@
-<?php
```

Comment:
Please restore the tests until the pro will be moved to the v2

Reply:
Handled https://github.com/elementor/elementor/pull/14205/commits/401180279f6ddbcb71fc3ca70c327dd33a821438